Sources/MapboxDirectionsCLI/CodingOperation.swift

```diff
@@ -18,7 +30,8 @@ class CodingOperation<ResponceType : Codable, OptionsType : DirectionsOptions >
         return try encoder.encode(result)
```

Comment:
The result here is exactly `directionsResultsProvider` needed below. We can return a tuple `(Data, ResponceType)` here (and also remove generic parameter as `T` is always `ResponceType` here.